.github/scripts/build.sh

```diff
@@ -51,7 +51,7 @@ fi
 
 $ROOT_PATH/.github/scripts/toolchain/build-gcc.sh
 
-if [[ "$PLATFORM" =~ (mingw|cygwin) ]]; then
+if [[ "$PLATFORM" =~ mingw ]]; then
```

Comment:
As discussed with MinGW community there is no use case for building entire MinGW with Cygwin toolchain, especially `winpthread` library is not compatible with Cygwin system headers. Only MinGW CRT is needed to build Cygwin GCC which is done in `build-mingw-crt.sh`.